### PR TITLE
chore: update gradle wrapper

### PR DESCRIPTION
**Pull request by kozaxinan**
Fixes #

Called "./gradlew wrapper" to update gradle wrapper for the current version.

gradle commands should work
.

## Checklist

- [ ] Documented
- [ ] Unit tested
- [ ] Integration tests updated

### CODE REVIEW

**Comment by Fry-kun** [comment]
@flank-it

**Comment by kozaxinan** [comment]
@flank-it

File: gradlew.bat
Comment (Fry-kun) [inline gradlew.bat:1]: Seems to be using LF for line endings. Pretty sure Windows prefers CRLF.

**Review by Fry-kun — CHANGES_REQUESTED** [review]

File: gradlew.bat
Comment (kozaxinan) [reply gradlew.bat:1]: Screenshot 2023-04-28 at 10 56 33

It has windows line endings.

Comment (Fry-kun) [reply gradlew.bat:1]: Sg, my diff probably confused me

**Review by Fry-kun — APPROVED** [review]

**Pull request merged**